packages/webpack-module-federation/src/defineConfig.ts

```diff
@@ -60,6 +60,18 @@ export interface DefineHostModuleFederationPluginOptions extends ModuleFederatio
     router?: Router;
 }
 
+const useNamedChunkIdsTransformer: WebpackConfigTransformer = (config: WebpackConfig) => {
```

Comment:
are you really attached to this name? prefixing a function with use makes it weird at first sight, expecting a react hook

Reply:
Updated for `setNamedChunkIdsTransformer`.